src/objects.js

```diff
@@ -1,42 +1,53 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+    name,
+    age
+  };
+return person;
+
 };
 
 const getName = object => {
-  // your code here
+return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(person => person.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  return people.reduce((acc, person) => acc + person.age, 0) / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
-};
+  const person = {
+    name,
+    age,
+    introduce: stranger => {
+      return `Hi ${stranger}, my name is ${person.name} and I am ${person.age}!`;
+    }
+  };
+  return person;};
```

Comment:
The closing curly braces }, is on the wrong line here, press Enter after person; 

Also you could just directly return this object, instead of assigning it to a variable as it is not used anywhere else, same as in the first function _createPerson_